runtime/tools/python/ttrt/__init__.py

```diff
@@ -20,6 +21,25 @@
 from ttrt.common.api import API
 
 
+def get_ttrt_metal_home_path():
+    package_name = "ttrt"
+    spec = importlib.util.find_spec(package_name)
+    package_path = os.path.dirname(spec.origin)
+    tt_metal_home = f"{package_path}/runtime"
+    return tt_metal_home
+
+
+os.environ["TT_METAL_HOME"] = get_ttrt_metal_home_path()
+
+new_linker_path = f"{get_ttrt_metal_home_path()}/tests"
+current_ld_library_path = os.environ.get("LD_LIBRARY_PATH", "")
+if current_ld_library_path:
+    updated_ld_library_path = f"{new_linker_path}:{current_ld_library_path}"
+else:
+    updated_ld_library_path = new_linker_path
+os.environ["LD_LIBRARY_PATH"] = updated_ld_library_path
+
+
```

Comment:
thank you for this change!!! Can we move this below all the system imports and before any custom imports (ie 
```
import ttrt.binary
from ttrt.common.api import API
```

I think it won't work like this

Reply:
Oh, that's a good point, doing now